src/App.tsx

```diff
@@ -1,35 +1,273 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import React, { useEffect, useState } from "react";
+import "./App.css";
 
-function App() {
-  const [count, setCount] = useState(0)
+interface Block {
+  id: number;
+  firstTeam: string;
+  secondTeam: string;
+  tickets: string;
+  stadium: string;
+}
+
+const App: React.FC = () => {
+  const [firstTeam, setFirstTeam] = useState<string>("");
+  const [secondTeam, setSecondTeam] = useState<string>("");
+  const [firstTeamDirty, setFirstTeamDirty] = useState<boolean>(false);
+  const [secondTeamDirty, setSecondTeamDirty] = useState<boolean>(false);
+  const [firstTeamError, setFirstTeamError] = useState<string>(
```

Comment:
As for me, it is more consistent to place error text under the textfield and make it smaller. What do you think?
![image](https://github.com/proga-react-practice/sigma-bohdan-mylyi/assets/74414048/b17d829d-2f2e-41a1-b190-62f8ff216171)